docs/templates/modules/README.md

```diff
@@ -0,0 +1,16 @@
+# FinOps toolkit modules
+
+FinOps toolkit templates are comprised of [Bicep](https://learn.microsoft.com/azure/azure-resource-manager/bicep) modules. Bicep is a domain-specific language that uses declarative syntax to define and deploy Azure resources. For a guided learning experience, start with the [Fundamentals of Bicep](https://learn.microsoft.com/training/paths/fundamentals-bicep/).
```

Comment:
Since the custom deployment seems to be limited to json should we talk about Bicep?

Reply:
Bicep generates JSON. We should be good.